simulation-runner/config/scenarios/ops-scenario.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This file is likely incomplete and will need to be updated once we're able to get the scala simulation further in its execution.

We'll probably need to change the `num_instances`, etc values to something more interesting for our OPS job.